pkg/cmd/roachtest/spec/cluster_spec.go

```diff
@@ -125,6 +125,7 @@ type ClusterSpec struct {
 		MachineType    string
 		MinCPUPlatform string
 		VolumeType     string
+		VolumeCount    int
```

Comment:
Super nit / optional: Might be worthwhile to add a comment here saying that volume count is only supported on GCE currently, but if any other clouds start supporting it the variable should be lifted up.